ceremonies/src/lib.rs

```diff
@@ -1831,7 +1839,7 @@ impl<T: Config> Pallet<T> {
 		cindex: CeremonyIndexType,
 		meetup_index: MeetupIndexType,
 		meetup_participants: &[T::AccountId],
-		attestations: &[T::AccountId],
+		attestations: &BoundedVec<T::AccountId, T::MaxAttestations>,
```

Comment:
I think you can leave this as it is. it has nothing directly to do with the storage value.

Reply:
f5b8a8af5efa494f0bbd43e059785f94b4eebadd